src/main/resources/application.yaml

```diff
@@ -1,4 +1,6 @@
 spring:
+  liquibase:
+    change-log: db/changelog/db.changelog-master.yaml
   sql:
     init:
       mode: always
```

Comment:
Now that the database schema is managed by Liquibase, you have to modify all the previous configuration from Spring